common/functions/src/scalars/uuids/uuid_creator.rs

```diff
@@ -91,7 +91,7 @@ where T: UUIDCreator + Clone + Sync + Send + 'static
         Ok(DataType::String)
     }
 
-    fn nullable(&self, _input_schema: &DataSchema) -> Result<bool> {
+    fn nullable(&self, _arg_fields: &[DataField]) -> Result<bool> {
```

Comment:
If the arg size is zero, it's zero not null, so this method can be removed?

Reply:
Yes, it should be removed.